src/NodeComponent.ts

```diff
@@ -86,4 +95,13 @@ export abstract class NodeComponent<T extends Node> {
 		slotNode.replaceWith(this.node);
 		return this;
 	}
+
+	/**
+	 * Empty the component (remove children)
+	 */
+	public empty() {
+		while (this.node.firstChild)
+			this.node.removeChild(this.node.firstChild);
+		return this;
+	}
```

Comment:
I need to check if `replaceChildren` exists on `Node`. If it does, this can be implemented without the `if`s. If not, `replaceChildren` cannot be used.

Reply:
forget the memory